frontend/src/metabase/visualizations/echarts/cartesian/utils/timeseries.ts

```diff
@@ -249,10 +249,28 @@ export function getLargestInterval(intervals: TimeSeriesInterval[]) {
   });
 }
 
+const isTimezoneInOffsetFormat = (timezone: string) => {
```

Comment:
you could use regex groups and extract the sign, hours, and minutes here without parsing it again

Reply:
Thanks, I addressed that